### PR TITLE
Feature/kafka health

### PR DESCRIPTION
**Pull request by sergeysenja1992**

### CODE REVIEW

File: src/main/resources/config/application-dev.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ spring:
             discovery:
                 enabled: true
     kafka:
-        bootstrap-servers: localhost:9093
```
Comment (v-kyrychenko) [inline src/main/resources/config/application-dev.yml]: why port changed?

File: src/main/resources/config/application.yml [hunk 2]
```diff
@@ -54,6 +54,8 @@ management:
     health:
         mail:
             enabled: false # When using the MailService, configure an SMTP server and set this to true
+        binders:
```
Comment (v-kyrychenko) [inline src/main/resources/config/application.yml]: does it work with dev profile, where kafka not used?

**Review by v-kyrychenko — APPROVED** [review]

File: src/main/resources/config/application-dev.yml (hunk 1 above)
Comment (sergeysenja1992) [reply src/main/resources/config/application-dev.yml]: because default kafka port is 9092

File: src/main/resources/config/application.yml (hunk 2 above)
Comment (sergeysenja1992) [reply src/main/resources/config/application.yml]: yes, you are right, my fault

File: src/main/resources/config/application-dev.yml (hunk 1 above)
Comment (v-kyrychenko) [reply src/main/resources/config/application-dev.yml]: in all our MS default value - 9093
i think such changes must be done for whole services after conversation
for some reason jhipster generate exactly 9093

**Review by dtsarenko — APPROVED** [review]

**Pull request merged**